graphics/src/AssimpLoader.cc

```diff
@@ -69,10 +69,12 @@ class AssimpLoader::Implementation
   /// \param[in] _scene the assimp scene
   /// \param[in] _matIdx index of the material in the scene
   /// \param[in] _path path where the mesh is located
+  /// \param[in] _fileBaseName Base name of the mesh file.
   /// \return pointer to the converted common::Material
-  public: MaterialPtr CreateMaterial(const aiScene* _scene,
+  public: MaterialPtr CreateMaterial(const aiScene *_scene,
                                      unsigned _matIdx,
-                                     const std::string& _path) const;
+                                     const std::string &_path,
+                                     const std::string &_fileBaseName) const;
```

Comment:
This is breaking API in `gz-common5`, you should target `main`, isn't it ?

Reply:
this function is in the private `AssimpLoader::Implementation` class and not exposed in header. So should be fine?